src/xdocs/report_issue.xml

```diff
@@ -155,14 +155,49 @@ expected level should be 12. [Indentation]
       </p>
 
       <p>
-        Please provide detailed description of problem and chunks of code and config
-        that you currently have.
+        Please provide a clear and detailed description of the problem and describe what
+        you would want to happen instead. Relevant chunks of code and config that you currently
+        have will definitely facilitate understanding. Any other context or
+        screenshots which elaborate further on the relevance and rational behind
+        the new feature may also be added.
       </p>
 
       <p>
         Imagine that everything is possible and propose name of the new option and its behaviour.
         Do not think that your issue is so obvious.
       </p>
+
+      <p>
+        Imagine that everything is possible and propose name of the new option and its
+        describe its behaviour.
+        Never think that your issue is obvious.
+      </p>
```

Comment:
please remove this block , it is the same as previos

Reply:
done